projects/canopy/src/lib/progress-indicator/progress-indicator.module.ts

```diff
@@ -0,0 +1,23 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+import { LgHeadingModule } from '../heading';
+
+import { LgProgressIndicatorComponent } from './progress-indicator.component';
+import { LgProgressBarComponent } from './progress-bar/progress-bar.component';
+import { LgProgressHeaderComponent } from './progress-header/progress-header.component';
+import { LgProgressJourneyComponent } from './progress-journey/progress-journey.component';
+
+const components = [
+  LgProgressIndicatorComponent,
+  LgProgressBarComponent,
+  LgProgressHeaderComponent,
+  LgProgressJourneyComponent,
+];
+
+@NgModule({
+  imports: [ CommonModule, LgHeadingModule ],
+  declarations: [ components ],
+  exports: [ components ],
+})
+export class LgProgressIndicatorModule {}
```

Comment:
Please can we make this standalone in the first instance? We want the merge of `master` into `next` to be seamless, not to include further work, as the conversion to Standalone components was already huge.

Reply:
I'm happy either way with this, but my first thought was that we probably shouldn't make this standalone in the first instance as it means that when this feature is merged, there will be a version of Canopy released with a strange inconsistency in it compared with all the other components. Perhaps that's not important in the grand scheme of things but thought I'd throw it out there as it's an open source library. What do you think @pauleustice?

Reply:
@jaynguyenslng I don't think you can export standalone components from a module so I think we need to delete the module. It _should_ be throwing a compile error when trying to use the module (you could test it out in the test app)

Reply:
To address @owensgit's point about it being an anomaly with the use of Standalone, perhaps we could signal that on the story/docs for the component so that it's clear?